glide.yaml

```diff
@@ -1,18 +1,25 @@
 package: github.com/edgexfoundry/device-sdk-go
 import:
 - package: github.com/edgexfoundry/edgex-go
+  version: delhi
   subpackages:
   - /pkg/models
   - /pkg/clients/coredata
   - /pkg/clients/metadata
   - /pkg/clients/logging
 - package: github.com/BurntSushi/toml
+  version: a368813c5e648fee92e5f6c30e3944ff9d5e8895
 - package: github.com/gorilla/mux
+  version: =1.3.0
```

Comment:
Using this gorilla/mux version, `make test` fails:
```
# github.com/edgexfoundry/device-sdk-go/internal/controller
internal/controller/restfuncs_test.go:77:8: undefined: mux.SetURLVars
internal/controller/restfuncs_test.go:106:8: undefined: mux.SetURLVars
```
Using version 1.6.2 the issue is fixed.

Reply:
Thanks, @jduranf 
I've changed it to 1.6.2
The unit test on my IDE was passed, so I didn't notice that